<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>470</width>
    <height>133</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <widget class="QLabel" name="chooseDataFileLabel">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Choose data file:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="dataFileLocationCBox">
       <item>
        <property name="text">
         <string>Local RDF File(s)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>RDF Resource from URI</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="startValidationButton">
       <property name="text">
        <string>Start Validation</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="chosenValidatorFile">
       <item>
        <property name="text">
         <string>Geometry Literal Content Validation</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>GeoSPARQL 1.0 Validation</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>GeoSPARQL 1.1 Validation</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>GeoSPARQL Extended Rule Set Validation</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>TTL Validator</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="dataFileLocationLabel">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Data File Location:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Chosen Validation Rule Set:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QgsFileWidget" name="validationFileWidget">
       <property name="storageMode">
        <enum>QgsFileWidget::GetMultipleFiles</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsFileWidget</class>
   <extends>QWidget</extends>
   <header>qgsfilewidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>314</x>
     <y>111</y>
    </hint>
    <hint type="destinationlabel">
     <x>234</x>
     <y>66</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
